lib/__tests__/__snapshots__/transform.js.snap

```diff
@@ -240,3 +240,339 @@ foo
 </span>
 =========="
 `;
+
+exports[`native components basic 1`] = `
+"==========
+
+      export default class FooComponent extends Component {
+      }
+  
+~~~~~~~~~~
+foo
+~~~~~~~~~~
+ => tagName: div
+~~~~~~~~~~
+
+      import { tagName } from \\"@ember-decorators/component\\";
+      @tagName(\\"\\")
+      export default class FooComponent extends Component {
+      }
```

Comment:
I'd prefer: 

```js
export default class FooComponent extends Component {
  tagName = '';
}
```

since it doesn't need the additional addon.

Reply:
Yes, I was also unsure at the beginning. But `ember-native-class-codemod` does also transform components to use the decorator (see https://github.com/ember-codemods/ember-native-class-codemod/blob/master/transforms/ember-object/__testfixtures__/decorators.output.js#L33), and we need the other decorators from the addon as well in most cases, so I thought we should align this codemod with what `ember-native-class-codemod` does. What do you think?

Reply:
I might be wrong but I think at first tagName did not work out of the box and the decorator was actually required which might be why the other codemod is still using it. maybe @pzuraq can give us an opinion on this.

Reply:
Asked on Discord. @pzuraq replied that decorator should be used. Hope [this direct link to message](https://discordapp.com/channels/480462759797063690/608346628163633192/672126515559006241) is working.

Reply:
hmm, I'm still not a fan, but okay. I think in that case we should check if the dependency exists, and if not offer to install it. I guess we could keep track of whether we find a native class that should be migrated and only run the install code afterwards if we had found one. does that make sense?

Reply:
Honestly, I think it should be sufficient to mention running `ember install ember-decorators` if required. That's what `ember-native-class-codemod` is doing [here](https://github.com/ember-codemods/ember-native-class-codemod#usage.

Given the limited amount of cycles we have available, and the work it requires to implement that, I think the time is spent much better on other issues, that yield more value. At least I would not be willing/able to invest the time, sorry! 🤷‍♂️

Reply:
while I would personally like to see https://github.com/emberjs/rfcs/issues/589 land to resolve this, I guess we shouldn't block the PR on that and can still rework this later if that RFC issue is implemented at some point.